packages/runner-ct/lib/runner-ct.ts

```diff
@@ -59,3 +59,10 @@ export const serve = (req, res, options: ServeOptions) => {
     projectName: config.projectName,
   })
 }
+
+export const serveChunk = (req, res, options) => {
+  let { config } = options
+  let pathToFile = getPathToDist(req.originalUrl.replace(config.clientRoute, ''))
+
+  return send(req, pathToFile).pipe(res)
+}
```

Comment:
Very clever.

Could you comment on what this function does?
Or maybe add a unit/integration test for it?

Reply:
I am sorry, but what is not understandable in the `serveChunk` name? Maybe instead of comment we can name this function so it will be clear what it does? 

Any ideas of a better name? thx